packages/perseus/src/components/svg-image.tsx

```diff
@@ -240,10 +241,7 @@ class SvgImage extends React.Component<Props, State> {
             labelDataIsLocalized: false,
             labels: [],
             labelsRendered: {},
-            range: [
-                [0, 0],
-                [0, 0],
-            ],
+            range: [[0, 0], [0, 0] as [number, number]],
```

Comment:
Just curious about this one. Was TS interpreting these as having to be zeros? Like `[0, 0]`? All these things called `range` is throwing me off, but I'm wondering if we could have a shared type for these.

Reply:
Those literals are inferred as `[]number` (unbounded arrays).  So the `as` makes it explicit that they're 2-item tuples.